src/services/chris_integration.tsx

```diff
@@ -108,48 +122,21 @@ class ChrisIntegration {
     return count;
   }
 
-  // old code used for manually uploaded image analysis
-  static async processNewAnalysis(files: LocalFile[]): Promise<boolean> {
```

Comment:
I would suggest leaving this function commented out instead of deleted even though it is not currently used because we may need it later. @janakitti whats your opinion about that?

Reply:
I think I agree. This method is used for a manual file upload feature, and there might be a chance that it gets added in again later on, as a third method for performing an analysis.